doc/main/tbb_userguide/snippets/flow_graph_examples.cpp

```diff
@@ -0,0 +1,76 @@
+/*
+    Copyright (c) 2005-2022 Intel Corporation
```

Comment:
The file did not exist before 2022. I am not sure whether we need to do it, but consider indicating that fact in the Copyright line. 
```suggestion
    Copyright (c) 2022 Intel Corporation
```